Gems/Atom/RHI/Vulkan/Code/Source/RHI/CommandQueueContext.cpp

```diff
@@ -383,6 +383,7 @@ namespace AZ
 
         CommandQueue& CommandQueueContext::GetPresentationCommandQueue() const
         {
+            AZ_Assert(m_presentationQueueIndex < aznumeric_cast<uint32_t>(-1), "Invalid presentation queue index");
```

Comment:
nit: It'd be nice to put `aznumeric_cast<uint32_t>(-1)` in a const variable called `InvalidIndex`, that way it can be assigned to `m_presentationQueueIndex` in the header and in here rather than doing a `<`, it'd be more suitable to do `m_presentationQueueIndex != InvalidIndex`